packages/vitest/src/runtime/moduleRunner/moduleRunner.ts

```diff
@@ -15,7 +15,8 @@ export class VitestModuleRunner extends ModuleRunner {
   public mocker: VitestMocker
   public moduleExecutionInfo: ModuleExecutionInfo
 
-  constructor(private options: VitestModuleRunnerOptions) {
+  constructor(private vitestOptions: VitestModuleRunnerOptions) {
```

Comment:
The base class `ModuleRunner` also has `options` property, which later accessed for `this.options.createImportMeta`
https://github.com/vitejs/vite/blob/84079a84ad94de4c1ef4f1bdb2ab448ff2c01196/packages/vite/src/module-runner/runner.ts#L342-L343. To avoid crashing the properly, I needed to rename this.